synapse/storage/databases/main/event_federation.py

```diff
@@ -1171,6 +1171,35 @@ def _get_min_depth_interaction(
 
         return int(min_depth) if min_depth is not None else None
 
+    async def has_room_extremities_changed_since(
```

Comment:
Suggest the name `have_room_forward_extremities_changed_since` to make it clear this doesn't care about backwards extremities